source/scss/themes/default/_theme.variables.scss

```diff
@@ -43,6 +43,7 @@ $c-quinary: $c-taxicab-yellow;
 $c-senary: $c-dusk-blue;
 $c-septenary: $c-brown-grey;
 $c-octonary: $c-dirty-salmon;
+$c-nonary: $c-purple;
```

Comment:
why not just use `$c-sponsor-color` for the places you're using `$c-nonary`?

Reply:
Good call. I forgot that was there...